ocaml/typing/jkind_axis.mli

```diff
@@ -0,0 +1,108 @@
+(**************************************************************************)
```

Comment:
This file is a bit heavyweight for just `typemodifier.ml`. But I am working on another PR that utilizes it within `jkind.ml`.